test/Model/mutators.js

```diff
@@ -0,0 +1,23 @@
+const {expect} = require('chai');
+const {RootModel} = require('../../lib/Model');
+
+describe('mutatotrs', () => {
```

Comment:
Also have a small typo here too:
```suggestion
describe('mutators', () => {
```